services/data/src/engine/types/FetchError.ts

```diff
@@ -1,17 +1,17 @@
 export type FetchErrorTypeName = 'network' | 'unknown' | 'access' | 'aborted'
-export type FetchErrorDetails = Response | Error
+export type FetchErrorDetails = object
```

Comment:
Following along from the comment above, I think we should be super-explicit here and use `export type FetchErrorDetails = { message?: string }`

Reply:
This will also allow someone catching a `FetchError` object to know (statically or with IDE auto-completion) that there is a possibly-undefined string property `message` on the `details` object, which is correct 👍

Reply:
See https://github.com/dhis2/app-runtime/pull/510#discussion_r412846708